src/data/CellarHomeDataMap.ts

```diff
@@ -0,0 +1,22 @@
+import { config } from "utils/config"
+import { CellarDataMap } from "./types"
+import { aaveStable, ethBtcTrend, ethBtcMomentum } from "./strategies"
+
+interface CellarHomeDataMap {
+  automatedPortofolio: CellarDataMap
+  yieldStrategies: CellarDataMap
+}
+
+let cellarHomeConfig: CellarHomeDataMap
```

Comment:
We don't want to have too many source data constant, let's move this to `cellarDataMap` and add a key `type` with enum `automatedPortfolio` and `yieldStrategies` there and in `PageHome` we show it with filtering the `type`.